pkgs/top-level/aliases.nix

```diff
@@ -555,6 +555,7 @@ mapAliases ({
   glide = throw "'glide' has been removed as it is unmaintained, please use Go modules instead"; # Added 2023-12-26
   glfw-wayland = glfw; # Added 2024-04-19
   glfw-wayland-minecraft = glfw3-minecraft; # Added 2024-05-08
+  globalprotect-openconnect = throw "'globalprotect-openconnect' has been renamed to/replaced by 'gpauth' and 'gpconnect'"; # Added 2024-09-21
```

Comment:
I assume this line meant to say 'gpclient' rather than 'gpconnect'? Thanks for making this commit.